app/models/customer.rb

```diff
@@ -1,17 +1,19 @@
 class Customer < ApplicationRecord
   include PgSearch::Model
-  pg_search_scope :stemmed, ->(query, threshold) {
+  pg_search_scope :stemmed, lambda { |query, threshold|
     {
       query: query,
-      against: {name: "A", email: "B", company: "B", age: "B", status: "C"},
+      against: { name: 'A', email: 'B', bio: 'B', company: 'B', age: 'B', status: 'C' },
```

Comment:
Is there a reason that you switched from double to single quotes? Again, `standardrb` is not going to love this.